tutorials/foreign-object.html

```diff
@@ -0,0 +1,813 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+
+        <meta name="description" content="Create interactive diagrams in JavaScript easily. JointJS plugins for ERD, Org chart, FSA, UML, PN, DEVS, LDM diagrams are ready to use." />
+
+        <link rel="stylesheet" href="css/tutorial.css" />
+        <link rel="stylesheet" href="../node_modules/prism-themes/themes/prism-one-light.css">
+
+        <!-- Dependencies: -->
+        <script src="../node_modules/jquery/dist/jquery.js"></script>
+        <script src="../node_modules/lodash/lodash.js"></script>
+        <script src="../node_modules/backbone/backbone.js"></script>
+
+
+        <link rel="stylesheet" type="text/css" href="../build/joint.min.css" />
+        <link rel="stylesheet" href="css/foreign-object.css">
+        <script type="text/javascript" src="../build/joint.min.js"></script>
+
+        <title>JointJS - JavaScript diagramming library - Getting started.</title>
+    </head>
+    <body class="language-javascript tutorial-page">
+
+
+        <div id="foreign-object" class="tutorial">
+
+            <h1>Foreign Object</h1>
+
+            <p>
+                In the wonderful world of SVG elements, 
+                <a href="https://developer.mozilla.org/en-US/docs/Web/SVG/Element/foreignObject" target="_blank"><code>foreignObject</code></a> 
+                stands out from the rest as it provides some unique behaviour that other SVG elements do not. SVG, like other XML dialects, is
+                namespaced. That means if you want to include elements from a different XML namespace right in your SVG, you need some
+                mechanism to enable this functionality. That's where <code>foreignObject</code> comes into play.
+            </p>
+
+            <p>
+                How does this relate to creating diagrams with JointJS? While JointJS provides a lot of functionality and customization
+                for users, we as developers are a habitual bunch. If we want to create some basic interactive elements, we often want to
+                reach for technologies we are already familiar with, and in web land, the dominant force in this regard is still good old HTML.
+            </p>
+
+            <p>
+                Historically, recommending <code>foreignObject</code> to users of JointJS was a little bit tricky, the main reason for this was the
+                inherent lack of support for <code>foreignObject</code> in Internet Explorer. Due to the necessity of supporting all our users,
+                including those who were still using IE, there was a hesitancy about integrating support into the library fully. Luckily for us, 
+                the days in which we had to support IE are now at an end, and all other major browsers have full support which can be
+                seen in the following <a href="https://caniuse.com/?search=foreignObject" target="_blank">caniuse</a> reference table.
```

Comment:
replace ending to say "...full support, as illustrated by this caniuse reference table"